app/src/main/java/com/example/stardewvalleypockettracker/MainActivity.java

```diff
@@ -1,14 +1,34 @@
 package com.example.stardewvalleypockettracker;
 
-import androidx.appcompat.app.AppCompatActivity;
-
+import android.content.Intent;
+import android.media.Image;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends BaseActivity {
 
-    @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        setTitle(R.string.activity_main);
+        setSettingsVisibility(View.VISIBLE);
+
+        View mButton = findViewById(R.id.settings_icon);
+        mButton.setOnClickListener(new View.OnClickListener(){
+            @Override
+                    public  void onClick(View view){
+                       openSettingsMenu();
+            }
+        });
+        int IR = getResources().getIdentifier("drawable/testlogo", null, this.getPackageName());
```

Comment:
Use different casing here (all caps is usually for constants) and give it a more meaningful name